### PR TITLE
chore: Use QueryLogger any time we log a query #9835

### PR DESCRIPTION
**Pull request by wcarlson5**
### Description 
_What behavior do you want to change, why, how does your patch achieve the changes?_

### Testing done 
_Describe the testing strategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: ksqldb-parser/src/main/java/io/confluent/ksql/parser/DefaultKsqlParser.java [hunk 1]
```diff
@@ -86,9 +86,9 @@ public PreparedStatement<?> prepare(
 
       return PreparedStatement.of(stmt.getUnMaskedStatementText(), root);
     } catch (final ParseFailedException e) {
-      if (!e.getSqlStatement().isEmpty()) {
-        throw e;
-      }
+      //      if (!e.getSqlStatement().isEmpty()) {
+      //        throw e;
+      //      }
```
Comment (mjsax) [inline ksqldb-parser/src/main/java/io/confluent/ksql/parser/DefaultKsqlParser.java:91]: We should keep this actually. Cf https://github.com/confluentinc/ksql/pull/9842/commits/1289c837ce421300ec4bc96f2285def3495dcdcc

File: ksqldb-parser/src/test/java/io/confluent/ksql/parser/KsqlParserTest.java [hunk 2]
```diff
@@ -427,12 +427,6 @@ public void testSelectAll() {
     assertThat(query.getSelect().getSelectItems(), is(contains(new AllColumns(Optional.empty()))));
   }
 
-  @Test
-  public void testReservedColumnIdentifers() {
-    assertQuerySucceeds("SELECT ROWTIME as ROWTIME FROM test1 t1;");
-    assertQuerySucceeds("SELECT ROWKEY as ROWKEY FROM test1 t1;");
-  }
```
Comment (mjsax) [inline ksqldb-parser/src/test/java/io/confluent/ksql/parser/KsqlParserTest.java:432]: What's the issue with this test? Why remove it?

Comment (wcarlson5) [reply ksqldb-parser/src/test/java/io/confluent/ksql/parser/KsqlParserTest.java]: There was a Rowkey resolved issues that I resolved

File: ksqldb-parser/src/main/java/io/confluent/ksql/parser/DefaultKsqlParser.java (hunk 1 above)
Comment (wcarlson5) [reply ksqldb-parser/src/main/java/io/confluent/ksql/parser/DefaultKsqlParser.java]: Ah I see

**Review by mjsax — APPROVED** [review]

**Pull request merged**